checkstyle-configuration.xml

```diff
@@ -186,8 +204,8 @@
                 value="System.out.println." />
         </module>
 
-        <module name="Regexp">
-            <property name="illegalPattern" value="true" />
+        <module name="RegexpSinglelineJava">
+            <property name="ignoreComments" value="true" />
```

Comment:
Finally, if you add System.out.println and then comment it out, the build will pass.